ext/fluxnet_simulations/generic_site.jl

```diff
@@ -1 +1,169 @@
+"""
+    get_domain_info(FT, site_ID::Symbol; dz_bottom = FT(1.5), dz_top = FT(0.1),
+        nelements = 20, zmin = FT(-10),  zmax = FT(0))
 
+Gets and returns primary domain information for a generic Fluxnet site,
+using autofilled values from US-MOz (Missouri Ozark) site.
```

Comment:
I think here we can just say "using default values corresponding to a 10m deep soil column with 20 layers, with a resolution of 10 cm at the top of the domain and 1.5m at the bottom of the domain." (not tied to US-MOz at all)